### PR TITLE
Add options for config url, xml report and flagging alerts as INFO

### PR DESCRIPTION
**Pull request by psiinon**
Will also update the one in zap-api-python

### CODE REVIEW

**Comment by kingthorin** [comment]
Looks good

File: build/docker/zap-baseline.py [hunk 1]
```diff
@@ -120,7 +135,7 @@ def main(argv):
     usage()
     sys.exit(3)
     
-  if len(config) > 0 or len(generate) > 0 or len(report) > 0:
+  if len(config_file) > 0 or len(config_url) > 0 or len(generate) > 0 or len(report) > 0:
```
Comment (thc202) [inline build/docker/zap-baseline.py]: - Is the check of `config_url` really needed? (It does not seem to use the directory.)
 - The variable `report` is no longer defined, should be `report_html`/`report_xml` instead?

File: build/docker/zap-baseline.py [hunk 2]
```diff
@@ -270,12 +310,18 @@ def main(argv):
...
+      if len(report_html) > 0:
+        # Save the report
+        with open('/zap/wrk/' + report_html, 'w') as f:
+          f.write (zap.core.htmlreport())
+
+      if len(report_xml) > 0:
         # Save the report
-        with open('/zap/wrk/' + report, 'w') as f:
+        with open('/zap/wrk/' + report_xml, 'w') as f:
           f.write (zap.core.htmlreport())
```
Comment (thc202) [inline build/docker/zap-baseline.py]: Core call needs to be updated to XML report.

File: build/docker/zap-baseline.py (hunk 1 above)
Comment (psiinon) [inline build/docker/zap-baseline.py]: Ah yea, it should be report_html and report_xml (not config_url) as they are both written to the mapped drive

File: build/docker/zap-baseline.py (hunk 2 above)
Comment (psiinon) [inline build/docker/zap-baseline.py]: yep

**Comment by psiinon** [comment]
Hopefully fixed those

**Comment by thc202** [comment]
Thanks!

**Pull request merged**